src/app/onion/core/learning-object-builder.guard.ts

```diff
@@ -9,14 +9,6 @@ export class LearningObjectBuilderGuard implements CanActivate {
   constructor(private learningObjectService: LearningObjectService) { }
 
   canActivate(next: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> | Promise<boolean> | boolean {
-
-    return this.learningObjectService.getLearningObject(next.paramMap.get('learningObjectId')).then(learningObject => {
-      if (learningObject.status && learningObject.status === LearningObject.Status.RELEASED) {
-        return false;
-      }
-      return true;
-    }).catch(error => {
-      return false;
-    })
+    return true;
```

Comment:
If we're not using this Guard anymore can we delete it?

Reply:
Guard is now deleted.